package.json

```diff
@@ -28,7 +28,9 @@
         "json2csv": "^4.5.0",
         "lodash": "^4.17.11",
         "md5.js": "^1.3.5",
+        "n": "^6.0.1",
         "node-fetch": "^2.3.0",
+        "node-gyp": "^5.0.3",
```

Comment:
Are you sure we need those dependencies for the project? It's quite uncommon, as libraries you added are usually a dependencies of our dependencies, so they are already in the `node_modules`

Reply:
I think that it might be related with Johanna problems with starting the project. But it could be deleted right now and Johanna could see if everything is working fine.

Reply:
Yes, I had some trouble at the beginning but it should work fine without those dependencies now :)

Reply:
Are those dependencies required?